nav_msgs/Path[] triedPaths
geometry_msgs/Vector3[] initialSearch
float64[] costValues
geometry_msgs/Vector3 oldFrame
int16 chosen
